src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = days * 40;
+
+  if (days < 3) {
+    return sum;
+  }
+
+  if (days >= 7) {
+    return sum - 50;
+  }
+
+  if (days >= 3) {
+    return sum - 20;
```

Comment:
The check for 'days >= 3' is redundant since it's guaranteed by the previous condition. The code will never reach this point if 'days' is less than 3. You can simplify the logic by removing this if statement.